examples/getAllAccounts.js

```diff
@@ -1,13 +1,13 @@
-const { getAllAccounts, Account, Vault } = require('..');
-const { loadStdlib } = require('@reach-sh/stdlib');
+const {getAllAccounts, Account, Vault} = require('..');
+const {loadStdlib} = require('@reach-sh/stdlib');
 
 (async () => {
   const account = new Account({
     mnemonic: 'lens sell....',
     network: 'LocalHost',
   });
   const VAULT_ID = 123;
-  const vault = new Vault({ id: VAULT_ID });
```

Comment:
Did the linter auto correct this? If it did, we should change it so it is the opposite styling.

Reply:
@AwolDes 
Please kindly review
Thanks